src/owner.rs

```diff
@@ -1,4 +1,31 @@
-//! Owner pattern
+//! Owner pattern implements methods to query, manage and transfer ownership
+//! of the contract.
+//!
+//! The account that owns the contract is called the "current" owner. A contract
+//! is always initialized with an owner account. The current owner can propose an
+//! ownership transfer to a different account. This proposed account is the
+//! "proposed owner". If the proposed owner accepts the transfer, it becomes
+//! the current owner. The current owner can also renounce ownership of the
+//! contract.
+//!
+//! Note: There is no way to recover ownership of a renounced contract.
+//!
+//! The pattern consists of methods in [`Owner`] and [`OwnerExternal`]. The
+//! latter exposes methods externally and can be called by other contracts.
+//! This [derive_macro](near_contract_tools_macros::Owner)
+//! derives default implementation both these traits.
+//!
+//! # Safety
+//! The default implementation assumes or enforces the following invariants.
+//! Violating assumed invariants may corrupt contract state and show unexpected
+//! behavior [UB]. Enforced invariants throw an error [ERR] but contract
+//! state remains intact.
+//!
+//! * [UB] The owner root storage slot is not used or modified. The default key is `~o`.
+//! * [ERR] Only the current owner can call [`Owner::renounce_owner`] and [`Owner::propose_owner`]
+//! * [ERR] Only the proposed owner can call [`Owner::accept_owner`]
+//! * [ERR] The external functions exposed in [`OwnerExternal`] call their
```

Comment:
This notation generates some errors in `cargo doc`:

```text
warning: unresolved link to `UB`
  --> src/owner.rs:21:15
   |
21 | //! behavior [UB]. Enforced invariants throw an error [ERR] but contract
   |               ^^ no item named `UB` in scope
   |
   = note: `#[warn(rustdoc::broken_intra_doc_links)]` on by default
   = help: to escape `[` and `]` characters, add '\' before them like `\[` or `\]`

warning: unresolved link to `ERR`
  --> src/owner.rs:21:56
   |
21 | //! behavior [UB]. Enforced invariants throw an error [ERR] but contract
   |                                                        ^^^ no item named `ERR` in scope
   |
   = help: to escape `[` and `]` characters, add '\' before them like `\[` or `\]`
[...]
```

Reply:
Fixed this. Using round brackets now.